osu.Game/Users/UserStatistics.cs

```diff
@@ -26,17 +26,22 @@ public struct LevelInfo
             public int Progress;
         }
 
-        [JsonProperty(@"pp")]
-        public decimal? PP;
+        [JsonProperty(@"rank")]
+        public UserRanks Ranks;
 
-        [JsonProperty(@"pp_rank")] // the API sometimes only returns this value in condensed user responses
-        private int? rank
+        // eventually UserRanks object will be completely replaced with separate global and country rank properties, see https://github.com/ppy/osu-web/blob/cb79bb72186c8f1a25f6a6f5ef315123decb4231/app/Transformers/UserStatisticsTransformer.php#L53.
+        // but for now, always point UserRanks.Global to the global_rank property, as that is included solely for requests like GetUsersRequest.
```

Comment:
I was initially planning to separate the user statistics class passed to `RulesetsStatistics` from this one, but it appears that `global_rank` is included in every user statistics response, so I decided to just put it here and replace it from the existing `pp_rank` property. (and the separation didn't look that sensible to me either way)